backend/kesaseteli/applications/tests/test_schools_api.py

```diff
@@ -0,0 +1,61 @@
+import pytest
+from rest_framework import status
+
+
+def get_schools_api_url():
+    return "/v1/schools/"
+
+
+@pytest.mark.django_db
+def test_schools_list_status_ok(api_client):
+    response = api_client.get(get_schools_api_url())
+    assert response.status_code == status.HTTP_200_OK
+
+
+@pytest.mark.django_db
+def test_schools_list_returns_list(api_client):
+    response = api_client.get(get_schools_api_url())
+    assert type(response.json()) == list
+
+
+@pytest.mark.django_db
+def test_schools_list_returns_non_empty_list(api_client):
+    response = api_client.get(get_schools_api_url())
+    assert type(response.json()) == list
+    assert len(response.json()) > 0
+
+
+@pytest.mark.django_db
+def test_schools_list_returns_non_empty_string_list(api_client):
+    response = api_client.get(get_schools_api_url())
+    assert type(response.json()) == list
+    assert len(response.json()) > 0
```

Comment:
How about counting the element then comparing it to `Schools.objects.count()`?

Reply:
Taking this into account.